lib/src/test/java/io/ably/lib/test/realtime/RealtimePresenceTest.java

```diff
@@ -1508,7 +1537,7 @@ public void onPresenceMessage(PresenceMessage message) {
             leavePresenceWaiter.waitFor(ably1.options.clientId, Action.leave);
 
             /* Validate that,
-             *	- we received all actions
+             *- we received all actions
```

Comment:
This change (and a few others) add noise. And, actually, in this case I think reduces readability of the comment.

Reply:
I don't usually add unnecessary code. I think this was added as a part of ```checkstyle``` run.